tsl/src/fdw/relinfo.c

```diff
@@ -157,77 +157,92 @@ get_total_number_of_slices(Hyperspace *space)
 }
 
 /*
- * Fillfactor values are between 0 and 1. It's an indication of how much data is in the chunk.
+ * Estimate fill factor for the chunks that don't have ANALYZE statistics.
+ * Fill factor values are between 0 and 1. It's an indication of how much data is
+ * in the chunk, expressed as a fraction of its maximum size.
```

Comment:
"maximum size" is a bit misleading here as there is really no maximum.

```suggestion
 * in the chunk, expressed as a fraction of its estimated final size.
```